### DISCUSSION TITLE
How to be a Github Campus Advisors?

### DISCUSSION THREAD

**Discussion by randiproska** [community/community#43454] [GitHub Education]
Dear Github,

I am looking for the application form for Github Campus Advisors and I don't found it. I read entirely about it here https://github.com/github-campus-advisors/Campus-Advisor-Training and here https://github.blog/2020-12-10-introducing-the-new-campus-advisors-program/.

However, I don't find the application form. Can anyone guide me how to go through the Github Campus Advisors process? Which steps should I involved in? Thanks.

Best regards,

**Comment by melvincwng** [comment]
Hey @randiproska, based on what I've saw from the previous (now defunct) GitHub Education Community forums over [here](https://education.github.community/t/problem-campus-advisor/73063) & the previous GitHub Campus Advisors application form over [here](https://airtable.com/shrIOpQrC8JJ7zsKM), it seems the like GitHub Campus Advisors program is **currently closed** at the moment.

As to when will the program reopen for applications, my advice would be to contact GitHub Education Support directly over [here](https://support.github.com/contact/education).

Hope this clarifies & answers your question. Take care, good luck 😄
